libmscore/slur.cpp

```diff
@@ -33,6 +33,10 @@ void SlurSegment::draw(QPainter* painter) const
       QPen pen(curColor());
       qreal mag = staff() ? staff()->mag(slur()->tick()) : 1.0;
 
+      QVector<qreal> dotted     = { 0.01, 1.99 }; // tighter than Qt Dotline equivalent { 0.01, 2.99 }
+      QVector<qreal> dashed     = { 3.00, 3.00 };
```

Comment:
I wonder whether here too we'd want to increase the gaps by the width to compensate for the SquareCap.
Just like you apparently do for the wide dashed line below

Reply:
The default Qt::DashLine is { 4.0, 2.0 }. To compensate for caps I changed it to { 3.0, 3.0 }  already. 
(I had first declared as { 3.01, 2.99 } but that felt a little too geeky since there are no dots involved.  If it makes things clear I can do that).